defold-spine/src/comp_spine_model.cpp

```diff
@@ -477,7 +477,6 @@ namespace dmSpine
                     SendAnimationDone(component, state, entry, event);
 
                     // The animation has ended, so we won't send any more on this
-                    ClearCompletionCallback(&track);
```

Comment:
Should we clear the function ref? E.g.
`track->m_AnimationCallbackRef = 0;`

Reply:
Ah, yes, this is good point. thx